irqstat

```diff
@@ -67,12 +67,14 @@ def gen_numa(numafile):
                 print("NUMACTL ERROR:")
                 print(error)
                 exit(1)
+
+            output = output.decode("utf8").split("\n")
```

Comment:
Looks good, but rather than decoding the output, another option would be to set encoding in the Popen call via the encoding, text, or universal_newlines parameter.

Setting 'universal_newlines=True' in the Popen call has the advantage of being compatible with Python 2.  That will make the output strings instead of binary data.

Reply:
Ok, let me try to use that then